source/about/maximize-microsoft-investment.rst

```diff
@@ -1,21 +1,52 @@
 Maximize Your Microsoft Investment
 ===================================
 
-Supplement existing investments in Microsoft Teams, M365, and Entra ID with Mattermost for Microsoft Teams v10 for everyday collaboration with Mattermost’s specialized workflow platform for technical and operational teams needing advanced customization, toolchain integration, and deployment to segregated networks.
+Unlock the full potential of your Microsoft Teams, M365, and Entra ID investment with Mattermost. Designed for operational teams that require advanced customization, secure workflows, and seamless deployment on segregated networks, Mattermost enhances your existing Microsoft solutions for critical mission success.
+
+Mattermost & Microsoft
```

Comment:
Seeing this, I agree, I think it can be removed.

Reply:
Removed